### PR TITLE
Single Video page and Liked Page

### PR DESCRIPTION
**Pull request by SaurabhChawda**
Added Single Video page and Liked Page.
Screenshot (366)
Screenshot (367)

### CODE REVIEW

File: src/components/AsideCard/AsideCard.jsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import { useNavigate } from "react-router-dom";
+import { useUser, useData } from "../../Contexts/Index";
+export function AsideCard() {
+  const navigate = useNavigate();
+  const { data } = useData();
+  const {
+    state: { currentVideo },
+    dispatch,
+  } = useUser();
+  console.log(currentVideo);
```
Comment (fahd23) [inline src/components/AsideCard/AsideCard.jsx:11]: Remove this console log

**Review by fahd23 — APPROVED** [review]
Code looks Fine to me

File: src/components/AsideCard/AsideCard.css [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+}
+.aside-card--content-date {
+  margin-left: 0.5rem;
+}
+.aside-card--content-title {
+  font-size: var(--font-size-regular);
+  justify-self: flex-start;
+}
+.aside-card--time {
+  font-size: 0.8rem;
```
Comment (ShreyashMahajan) [inline src/components/AsideCard/AsideCard.css:46]: try to use px for less than 1rem

File: src/components/Index.js [hunk 3]
```diff
@@ -4,3 +4,5 @@ export { Sidebar } from "./Sidebar/Sidebar.jsx";
 export { SearchBar } from "./SearchBar/SearchBar.jsx";
```
Comment (ShreyashMahajan) [inline src/components/Index.js:4]: i liked that you created index file for exporting components

**Review by ShreyashMahajan — APPROVED** [review]
Code looks good and naming is also very understandable. Good to merge

File: src/components/AsideCard/AsideCard.jsx (hunk 1 above)
Comment (SaurabhChawda) [reply src/components/AsideCard/AsideCard.jsx:11]: Okay,ThankYou

**Pull request merged**